package/amazon-linux-packaging.spec

```diff
@@ -0,0 +1,149 @@
+%bcond_without check
+
+Name:           mount-s3
+Version:        1.20.0
+Release:        1.amzn2023
+Summary:        High-performance file system for Amazon S3
+
+# Main project is Apache-2.0, but bundled components have various licenses
+License:        Apache-2.0
+URL:            https://github.com/awslabs/mountpoint-s3
+Source0:        https://github.com/awslabs/mountpoint-s3/archive/v%{version}/mountpoint-s3-%{version}.tar.gz
+# Vendor tarball containing all Rust crate dependencies
+Source1:        mountpoint-s3-%{version}-vendor.tar.gz
+Source2:        LICENSE
+Source3:        NOTICE
+Source4:        THIRD_PARTY_LICENSES
+
+# BuildRequires lists dependencies that must be installed on the build system before the package can be compiled.
+BuildRequires:  clang
+BuildRequires:  clang-devel
+BuildRequires:  llvm
+BuildRequires:  rust
+BuildRequires:  cargo
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  git
+BuildRequires:  pkgconfig
+BuildRequires:  fuse-devel
+BuildRequires:  glibc-devel
+BuildRequires:  glibc-headers
+BuildRequires:  glibc-static
+BuildRequires:  libstdc++-devel
+BuildRequires:  nasm
+BuildRequires:  make
+BuildRequires:  which
+
+Requires:       fuse
+
+# BUNDLED C/C++ LIBRARIES - Required virtual provides for security tracking
+Provides: bundled(aws-c-auth)
+Provides: bundled(aws-c-cal)
+Provides: bundled(aws-c-common)
+Provides: bundled(aws-c-compression)
+Provides: bundled(aws-checksums)
+Provides: bundled(aws-c-http)
+Provides: bundled(aws-c-io)
+Provides: bundled(aws-c-s3)
+Provides: bundled(aws-c-sdkutils)
+Provides: bundled(aws-lc)
```

Comment:
I think we should specify exact version that we are bundling so that it's easier to track vulnerabilities.

E.g. `Provides: bundled(aws-lc) = 1.61.3`